src/test/resources/testdata/rest/joborder-data.txt

```diff
@@ -169,8 +169,8 @@
             },
             "source": "",
             "specialties": {
-                "total": 0,
-                "data": []
+                "total": 1,
+                "data": [12]
```

Comment:
think this might make the unit tests fail...data here should be an array of specialty JSON objects, like
[
{
  id: 12
},
{
  id: 13
}
]